report-generator/timestamp_analysis/post_processing_utils.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2023, RTE (http://www.rte-france.com)
```

Comment:
This is Thibault's script right?
If that's the case:
1. set him as author with `git commit --author "Thibault X <thibault.x@mail.com>"`
2. clearly state in the commit message that this commit was imported and if any modifications have been done, move them in a separate commit.